### PR TITLE
Abram

### PR DESCRIPTION
**Pull request by legoabram**

### CODE REVIEW

File: app/assets/javascripts/event-stage.js [hunk 1]
```diff
@@ -508,6 +518,7 @@
                         obj._dtid = eventStage.dataTables.uniqueId++;
                         var row = new self._row( { data: { row: obj } } ).$mount();
                         cache[ obj._dtid ] = row;
+                        debugger;
```
Comment (adamr501) [inline app/assets/javascripts/event-stage.js]: did you want this left in there?

File: app/assets/javascripts/event-stage.js [hunk 2]
```diff
@@ -24,6 +24,12 @@
             "feet": 3.28084,
             "meters": 1.0
         },
+        shorthand: {
+            "kilometers": 'km',
```
Comment (adamr501) [inline app/assets/javascripts/event-stage.js:28]: perfect use of double and single quotes on this. thanks!

File: app/views/event_staging/events/app.html.erb [hunk 3]
```diff
@@ -194,32 +194,36 @@
...
-							<td>{{ vm.row.distanceFromStart * units.forDistance() }}</td>
-							<td>{{ vm.row.vertGainFromStart * units.forElevation() }}</td>
-							<td>{{ vm.row.vertLossFromStart * units.forElevation()}}</td>
-							<td>
-								<span class="marker"><%= image_tag( "icons/dblue.svg", size: '16' ) %></span>
+							<td>{{ units.round( vm.row.distanceFromStart * units.forDistance(), 2 ) }}</td>
+							<td>{{ units.round( vm.row.vertGainFromStart * units.forElevation(), 2 ) }}</td>
+							<td>{{ units.round( vm.row.vertLossFromStart * units.forElevation(), 1 ) }}</td>
+							<td class="edit">
+								<!--<span class="marker"><%= image_tag( "icons/dblue.svg", size: '16' ) %></span>-->
```
Comment (adamr501) [inline app/views/event_staging/events/app.html.erb:212]: did you want this commented out code left in there?

File: app/assets/javascripts/event-stage.js (hunk 1 above)
Comment (legoabram) [reply app/assets/javascripts/event-stage.js]: WHOOPS. That's my biggest mistake in a while.

File: app/views/event_staging/events/app.html.erb [hunk 4]
```diff
@@ -327,21 +331,24 @@
...
+										<input type="text" class="form-control" v-model.number="model.longitude" v-mask="{ alias: 'decimal', digits: 6 }">
 									</div>
 									<div class="form-group col-xs-4">
 										<label>Elevation</label>
 										<span class="help-tooltip" tabindex="-1" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. ">
 											<i class="glyphicon glyphicon-question-sign"></i>
 										</span>
-										<input type="text" class="form-control" v-model="model.elevation" v-mask="{ alias: 'integer' }">
+										<div class="input-group">
+											<input-units type="text" class="form-control" v-model.number="model.elevation" v-mask="{ alias: 'integer' }" :scale="units.forElevation()"/>
```
Comment (adamr501) [inline app/views/event_staging/events/app.html.erb:349]: remove / from /> for html5 and i don't know what this is, but is this part correct? :scale="units.forElevation()"

**Pull request merged**